src/components/member/PasswordSetting.js

```diff
@@ -0,0 +1,228 @@
+    setCurrentPassword('');
+    setNewPassword('');
+    setConfirmPassword('');
+  };
+
+  const handleChangePassword = () => {
+    const checkingCurrentPassword = passwordValidator(currentPassword);
+    const checkingNewPassword = passwordValidator(newPassword);
+    const checkingConfirmPassword = passwordValidator(confirmPassword);
+    // verify empty fields
+    if (
+      user.get('password') &&
+      (checkingCurrentPassword ||
+        checkingNewPassword ||
+        checkingConfirmPassword)
+    ) {
+      setCurrentPasswordError(checkingCurrentPassword);
+      setNewPasswordError(checkingNewPassword);
+      setConfirmPasswordError(checkingConfirmPassword);
+    } else {
+      const checkingPasswordUpdate = newPasswordValidator(
+        currentPassword,
+        newPassword,
+        confirmPassword,
+      );
+      if (checkingPasswordUpdate) {
+        toast.error(checkingPasswordUpdate);
+      } else {
+        const checkingStrength = strengthValidator(newPassword);
+        if (checkingStrength) {
+          toast.error(checkingStrength);
+        } else {
+          onUpdatePassword({
+            id: userId,
+            password: newPassword,
+            currentPassword,
+          });
+          onClose();
+        }
+      }
+    }
+  };
+
+  const handleCurrentPasswordInput = (event) => {
+    setCurrentPassword(event.target.value);
+    setCurrentPasswordError(passwordValidator(event.target.value));
+  };
+  const handleNewPasswordInput = (event) => {
+    setNewPassword(event.target.value);
+    setNewPasswordError(passwordValidator(event.target.value));
+  };
+  const handleConfirmPasswordInput = (event) => {
+    setConfirmPassword(event.target.value);
+    setConfirmPasswordError(passwordValidator(event.target.value));
+  };
+
+  return (
+    <>
+      <Grid container spacing={3} className={classes.mainContainer}>
+        <Grid item xs={8}>
+          <Grid item xs={12}>
+            {user.get('password') !== null ? (
+              <Typography variant="h5">{t('Change Password')}</Typography>
+            ) : (
+              <Typography variant="h5">{t('Set Password')}</Typography>
+            )}
+          </Grid>
+          <Grid
+            container
+            spacing={3}
+            className={classes.changePasswordContainer}
+          >
+            <Grid item xs={12} sm={6}>
+              {user.get('password') !== null ? (
+                <TextField
+                  className={classes.input}
+                  required
+                  label={t('Current Password')}
+                  variant="outlined"
+                  value={currentPassword}
+                  error={currentPasswordError}
+                  helperText={currentPasswordError}
+                  onChange={handleCurrentPasswordInput}
+                  id={USER_CURRENT_PASSWORD_INPUT_ID}
+                  type="password"
+                />
+              ) : (
+                <Grid item xs={12} sm={6} />
```

Comment:
Why do you need these empty grid?
If you need to have a new line, you can nest `Grid` tags (with `container` rather than `item`)

Reply:
Got rid of it!